src/query-builder/InsertQueryBuilder.ts

```diff
@@ -262,6 +262,8 @@ export class InsertQueryBuilder<Entity> extends QueryBuilder<Entity> {
 
     /**
      * Adds additional ON CONFLICT statement supported in postgres and cockroach.
+     *
+     * @deprecated Use `orIgnore` or `orUpdate`
```

Comment:
why did this note is telling us to use `orUpdate` instead if `orUpdate` is marked as deprecated as well? I didn't get it

Reply:
Looks like only one of 3 overloads of `orUpdate` is deprecated.
Although I think [this deprecated overload](https://github.com/typeorm/typeorm/pull/7880/files/f69e9bdc67de469c586c538e3bb2a04c99667c98#diff-f12ef5527b9bc2971ab7a8247c71dd735a43cd7a1c859bd206b61dee8c7f617dR284) could just be removed instead - [new overload](https://github.com/typeorm/typeorm/pull/7880/files/f69e9bdc67de469c586c538e3bb2a04c99667c98#diff-f12ef5527b9bc2971ab7a8247c71dd735a43cd7a1c859bd206b61dee8c7f617dR284) will not break old code because it adds just one union type to first parameter.